pvlib/irradiance.py

```diff
@@ -88,7 +92,7 @@ def get_extra_radiation(datetime_or_doy, solar_constant=1366.1,
     # consider putting asce and spencer methods in their own functions
     method = method.lower()
     if method == 'asce':
-        B = solarposition._calculate_simple_day_angle(to_doy(datetime_or_doy))
+        B = (2. * np.pi / 365.) * (to_doy(datetime_or_doy))
```

Comment:
Alternative implementation: add an `offset` kwarg to `solarposition._calculate_simple_day_angle`:

```python
def _calculate_simple_day_angle(dayofyear, offset=1):
    """
    Calculates the day angle for the Earth's orbit around the Sun.

    Parameters
    ----------
    dayofyear : numeric
    offset : int, default 1
        choose 0 for the ASCE method or 1 for the Spencer method

    Returns
    -------
    day_angle : numeric
    """
    return (2. * np.pi / 365.) * (dayofyear - offset)
```

I'm fine with either approach

Reply:
I like your way better